source/prerequisites-aie-license.rst

```diff
@@ -5,12 +5,20 @@ AIE Build License
 
 To run any of the build time examples, you will need an AIE build license. Execute these instructions after the 'Full' install has completed.
 
-You will require the WSL instance MAC address in order to apply for a AIE build license. You can get your WSL MAC address by running the following command in Powershell:
+You will require the WSL instance MAC address in order to apply for a AIE build license. 
```

Comment:
It isn't WSL on Linux, right?

Reply:
Ah good catch, thanks @cathalmccabe, I'll update that now